src/Orleans/Orleans.csproj

```diff
@@ -74,8 +74,9 @@
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="Microsoft.CSharp" />
-    <Reference Include="Newtonsoft.Json">
-      <HintPath>$(SolutionDir)packages\Newtonsoft.Json.6.0.8\lib\net45\Newtonsoft.Json.dll</HintPath>
+    <Reference Include="Newtonsoft.Json, Version=6.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
```

Comment:
hmmm, I'm hesitant to constantly fight the tooling too. I understand it looks uglier than just having the short assembly names, but this does work well in NuGet (assuming you always upgrade references using NuGet and not try to bypass it). I can acknowledge this used to be an hassle in the pre-NuGet era, if you had to update a reference that was in several projects across several different solutions. This is no longer the case.

Having said that, it *might* soon be a non-issue if we end up migrating to kproj, due to (mostly @galvesribeiro 's) work to achieve CoreCLR compatibility.